alethzero/MainWin.cpp

```diff
@@ -232,6 +232,10 @@ Main::Main(QWidget *parent) :
 	{
 		ui->tabWidget->setTabText(0, ui->webView->title());
 	});
+	connect(ui->webView, &QWebEngineView::urlChanged, [=](QUrl const& _url)
+	{
+		ui->urlEdit->setText(_url.toString());
```

Comment:
For dapp packages urlEdit contains ethereum url (namereg path). The webview loads pages from http://localhost in such case and that url should stay hidden.